styles.css

```diff
@@ -447,3 +446,280 @@ input::placeholder {
 textarea::placeholder {
   color: #b3bac5;
 }
+
+/* To ignore for moblie */
+
+.desktop-header {
+  display: none;
+}
+
+.card-img1 {
+  display: none;
+}
+
+div.card-info1 {
+  display: none;
+}
+
+div.card-info2 {
+  display: none;
+}
+
+/* Desktop version */
+@media only screen and (min-width: 768px) {
+  .head-container {
+    margin: 0 155px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop-header {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 10px 0 10px 10px;
+    gap: 32px;
+  }
+
+  .nav {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .nav a {
+    text-decoration: none;
+  }
+
+  .main-sec {
+    display: block;
+    padding-left: 34px;
+    margin-top: 50px;
+  }
+
+  .main {
+    display: block;
+    background-image: url(./images/Header\ bg@2x.png);
+    height: 90vh;
+    width: 100%;
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 1px;
+  }
+
+  .wrapper {
+    margin: 200px 260px;
+  }
+
+  .welcome-note {
+    width: 400px;
+  }
+
+  .welcome-note h1 {
+    font-size: 48px;
+  }
+
+  .info {
+    font-size: 20px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    margin: 0 11%;
+    flex-shrink: 1;
+    gap: 40px;
+  }
+
+  div.card-img {
+    display: none;
+  }
+
+  div.card-img1 {
+    display: block;
+    background-size: contain;
+    align-self: center;
+  }
+
+  .card-info {
+    display: none;
+  }
+
+  div.card-info1 {
+    display: block;
+    margin: 24px 0 0 0;
+    width: 496px;
+  }
+
+  .tonic {
+    font-size: 40px;
+  }
+
+  .info-icons {
+    width: 306px;
+    margin: 0;
+  }
+
+  .info-icons h4 {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .info-text {
+    font-size: 16px;
+    width: 90%;
+  }
+
+  div.card-info2 {
+    display: block;
+    width: 496px;
+    margin: 24px 0 0 24px;
+    flex-shrink: 1;
+  }
+
+  .lang {
+    width: fit-content;
+  }
+
+  .rub {
+    flex-shrink: 0;
+  }
+
+  .button-div {
+    flex-shrink: 0;
+  }
+
+  .see-project1 {
+    padding: 12px 8px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    margin: 20px 0;
+    background: #4053fc;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  }
```

Comment:
Kindly implement the button interactions as given by the design.

Figma button interactions:
![image](https://user-images.githubusercontent.com/20554583/199766647-a7b88c7c-2183-4344-9e9e-388297b379e2.png)